mlrun/api/main.py

```diff
@@ -32,6 +34,20 @@
 app.include_router(api_router, prefix="/api")
 
 
+@app.exception_handler(mlrun.errors.HTTPStatusableError)
+async def http_statusable_error_handler(
+    request: Request, exc: mlrun.errors.HTTPStatusableError
+):
+    status_code = exc.response.status_code
+    error_message = repr(exc)
+    logger.warning(
+        'Failed handling request', error_message=error_message, status_code=status_code
```

Comment:
```suggestion
        'Request handling returned error status', error_message=error_message, status_code=status_code
```
The reason for this nit-picking is that some error statuses like 404 don't necessarily mean there's any "failure" in the code, technically, the request just failed, not the handling of it